### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by Dongwoo-Im** [Dongwoo-Im/short-paper-reivew#10]
Image
- 2025/03 [Scaling Vision Pre-Training to 4K Resolution (CVPR 2025)](https://arxiv.org/abs/2503.19903)
- 2024/11 [Analyzing The Language of Visual Tokens](https://arxiv.org/abs/2411.05001)
- 2024/11 [ALIT: Adaptive Length Image Tokenization via Recurrent Allocation (ICLR 2025)](https://arxiv.org/abs/2411.02393)
- 2024/04 [Editable Image Elements for Controllable Synthesis (ECCV 2024)](https://jitengmu.github.io/Editable_Image_Elements/)
- 2024/01 [pix2gestalt: Amodal Segmentation by Synthesizing Wholes (CVPR 2024 Hightlight)](https://gestalt.cs.columbia.edu/)
- 2023/12 [Amodal Completion via Progressive Mixed Context Diffusion (CVPR 2024 Highlight)](https://k8xu.github.io/amodal/)

VLM (MLLM)
- 2024/11 [Cut Your Losses in Large-Vocabulary Language Models (ICLR 2025 Oral)](https://arxiv.org/abs/2411.09009)
- 2024/11 [Mixture-of-Transformers: A Sparse and Scalable Architecture for Multi-Modal Foundation Models](https://arxiv.org/abs/2411.04996)
- 2024/10 [γ−MoD: Exploring Mixture-of-Depth Adaptation for Multimodal Large Language Models (ICLR 2025)](https://arxiv.org/abs/2410.13859)
- 2024/10 [CLEAR: Character Unlearning in Textual and Visual Modalities](https://arxiv.org/abs/2410.18057)
- 2024/09 [MM1.5: Methods, Analysis & Insights from Multimodal LLM Fine-tuning (ICLR 2025)](https://arxiv.org/abs/2409.20566)
- 2024/04 [BLINK: Multimodal Large Language Models Can See but Not Perceive (ECCV 2024)](https://zeyofu.github.io/blink/)
- 2024/03 [MagicLens: Self-Supervised Image Retrieval with Open-Ended Instructions (ICML 2024 Oral)](https://open-vision-language.github.io/MagicLens/)
- 2024/03 [Mini-Gemini: Mining the Potential of Multi-modality Vision Language Models](https://github.com/dvlab-research/MiniGemini)
- 2024/03 [MyVLM: Personalizing VLMs for User-Specific Queries (ECCV 2024)](https://snap-research.github.io/MyVLM/)
- 2024/03 [MM1: Methods, Analysis & Insights from Multimodal LLM Pre-training](https://arxiv.org/abs/2403.09611)
- 2024/01 [LLaVA-NeXT: Improved reasoning, OCR, and world knowledge](https://github.com/LLaVA-VL/LLaVA-NeXT)
- 2023/12 [FIND: Interfacing Foundation Models' Embeddings (NIPS 2024)](https://github.com/UX-Decoder/FIND)

Mamba
- 2024/10 [MatMamba: A Matryoshka State Space Model](https://arxiv.org/abs/2410.06718)
- 2024/03 [Cobra: Extending Mamba to Multi-Modal Large Language Model for Efficient Inference (AAAI 2025)](https://sites.google.com/view/cobravlm)
- 2024/03 [ZigMa: Zigzag Mamba Diffusion Model (ECCV 2024)](https://taohu.me/zigma/)
- 2024/03 [VideoMamba: State Space Model for Efficient Video Understanding (ECCV 2024)](https://github.com/OpenGVLab/VideoMamba)
- 2024/03 [LightM-UNet: Mamba Assists in Lightweight UNet for Medical Image Segmentation](https://github.com/MrBlankness/LightM-UNet)

Gen Image
- 2025/02 [RelaCtrl: Relevance-Guided Efficient Control for Diffusion Transformers](https://arxiv.org/abs/2502.14377)
- 2024/11 [SVDQunat: Absorbing Outliers by Low-Rank Components for 4-Bit Diffusion Models (ICLR 2025 Spotlight)](https://arxiv.org/abs/2411.05007)
- 2024/04 [Guiding a Diffusion Model with a Bad Version of Itself (NIPS 2024 Best Paper Runner-up Award)](https://arxiv.org/abs/2406.02507)
- 2024/04 [Paint by Inpaint: Learning to Add Image Objects by Removing Them First](https://arxiv.org/abs/2404.18212)
- 2024/04 [InstantFamily: Masked Attention for Zero-shot Multi-ID Image Generation](https://arxiv.org/abs/2404.19427)
- 2024/04 [Stylus: Automatic Adapter Selection for Diffusion Models (NIPS 2024 Oral)](https://stylus-diffusion.github.io/)
- 2024/04 [EdgeFusion: On-Device Text-to-Image Generation (CVPRW 2024)](https://arxiv.org/abs/2404.11925)
- 2024/04 [UniFL: Improve Stable Diffusion via Unified Feedback Learning (NIPS 2024)](https://uni-fl.github.io/)
- 2024/04 [CAN: Condition-Aware Neural Network for Controlled Image Generation (CVPR 2024)](https://arxiv.org/abs/2404.01143)
- 2024/04 [VAR: Visual Autoregressive Modeling: Scalable Image Generation via Next-Scale Prediction (NIPS 2024 Best Paper Award)](https://github.com/FoundationVision/VAR)
- 2024/04 [T-GATE: Cross-Attention Makes Inference Cumbersome in Text-to-Image Diffusion Models](https://github.com/HaozheLiu-ST/T-GATE)
- 2024/04 [Bigger is not Always Better: Scaling Properties of Latent Diffusion Models](https://arxiv.org/abs/2404.01367)
- 2024/03 [ReNoise: Real Image Inversion Through Iterative Noising (ECCV 2024)](https://garibida.github.io/ReNoise-Inversion/)
- 2024/02 [DistriFusion: Distributed Parallel Inference for High-Resolution Diffusion Models (CVPR 2024)](https://github.com/mit-han-lab/distrifuser)
- 2024/02 [Neural Network Diffusion](https://1zeryu.github.io/Neural-Network-Diffusion/)
- 2024/02 [InstanceDiffusion: Instance-level Control for Image Generation (CVPR 2024)](https://github.com/frank-xwang/InstanceDiffusion)
- 2024/01 [Mastering Text-to-Image Diffusion: Recaptioning, Planning, and Generating with Multimodal LLMs (ICML 2024)](https://github.com/YangLing0818/RPG-DiffusionMaster)
- 2023/09 [Finite Scalar Quantization: VQ-VAE Made Simple (ICLR 2024)](https://arxiv.org/abs/2309.15505)
- 2023/03 [Understanding Diffusion Objectives as the ELBO with Simple Data Augmentation (NIPS 2023 Oral)](https://arxiv.org/abs/2303.00848)
- 2022/12 [DiT: Scalable Diffusion Models with Transformers (ICCV 2023 Oral)](https://github.com/facebookresearch/DiT)

Video
- 2024/10 [MarDini: Masked Autoregressive Diffusion for Video Generation at Scale](https://mardini-vidgen.github.io/)
- 2024/03 [DINO-Tracker: Taming DINO for Self-Supervised Point Tracking in a Single Video (ECCV 2024)](https://dino-tracker.github.io/)
- 2024/03 [Be-Your-Outpainter: Mastering Video Outpainting through Input-Specific Adaptation (ECCV 2024)](https://be-your-outpainter.github.io/)
- 2024/03 [Mora: More like Sora for Generalist Video Generation](https://github.com/lichao-sun/Mora)
- 2024/02 [MC-ViT: Memory Consolidation Enables Long-Context Video Understanding](https://arxiv.org/abs/2402.05861)

3D
- 2024/06 [Grounding Image Matching in 3D with MASt3R](https://arxiv.org/abs/2406.09756)
- 2024/05 [SUNDAE: Spectrally Pruned Gaussian Fields with Neural Compensation](https://runyiyang.github.io/projects/SUNDAE/)
- 2024/04 [RefFusion: Reference Adapted Diffusion Models for 3D Scene Inpainting](https://reffusion.github.io/)
- 2024/03 [2DGS: 2D Gaussian Splatting for Geometrically Accurate Radiance Fields (SIGGRAPH 2024)](https://surfsplatting.github.io/)
- 2024/03 [RadSplat: Radiance Field-Informed Gaussian Splatting for Robust Real-Time Rendering with 900+ FPS](https://m-niemeyer.github.io/radsplat/)
- 2024/03 [MVEdit: Generic 3D Diffusion Adapter Using Controlled Multi-View Editing](https://lakonik.github.io/mvedit/)
- 2023/12 [DUSt3R: Geometric 3D Vision Made Easy (CVPR 2024)](https://dust3r.europe.naverlabs.com/)
- 2023/12 [Feature3DGS: Supercharging 3D Gaussian Splatting to Enable Distilled Feature Fields (CVPR 2024 Highlight)](https://feature-3dgs.github.io/)
- 2023/12 [SMERF: Streamable Memory Efficient Radiance Fields for Real-Time Large-Scene Exploration (SIGGRAPH 2024)](https://arxiv.org/abs/2312.07541)

Diffusion
- 2025/02 [NExT-Mol: 3D Diffusion Meets 1D Language Modeling for 3D Molecule Generation (ICLR 2025)](https://www.arxiv.org/abs/2502.12638)
- 2024/06 [Discrete Diffusion Modeling by Estimating the Ratios of the Data Distribution (ICML 2024 Best Paper)](https://arxiv.org/abs/2310.16834)

Efficinet LLM
- 2024/11 [BitNet a4.8: 4-bit Activations for 1-bit LLMs](https://arxiv.org/abs/2411.04965)
- 2024/10 [LoRA Done RITE: Robust Invariant Transformation Equilibration for LoRA Optimization (ICLR 2025 Oral)](https://arxiv.org/abs/2410.20625)
- 2024/07 [ThinK: Thinner Key Cache by Query-Driven Pruning (ICLR 2025 Spotlight)](https://arxiv.org/abs/2407.21018)
- 2024/06 [What Matters in Transformers? Not All Attention is Needed](https://arxiv.org/abs/2406.15786)
- 2024/04 [Better & Faster Large Language Models via Multi-token Prediction (ICML 2024)](https://arxiv.org/abs/2404.19737)
- 2024/04 [I-Design: Personalized LLM Interior Designer](https://atcelen.github.io/I-Design)
- 2024/04 [ReFT: Representation Finetuning for Language Models (NIPS 2024 Spotlight)](https://github.com/stanfordnlp/pyreft)
- 2024/03 [Transformer-Lite: High-efficiency Deployment of Large Language Models on Mobile Phone GPUs](https://arxiv.org/abs/2403.20041)
- 2024/03 [The Unreasonable Ineffectiveness of the Deeper Layers (ICLR 2025)](https://arxiv.org/abs/2403.17887)
- 2024/03 [EMMET: A Unified Framework for Model Editing (EMNLP 2024 Findings)](https://arxiv.org/abs/2403.14236)
- 2024/03 [PERL: Parameter Efficient Reinforcement Learning from Human Feedback](https://arxiv.org/abs/2403.10704)
- 2024/03 [GaLore: Memory-Efficient LLM Training by Gradient Low-Rank Projection (ICML 2024 Oral)](https://github.com/jiaweizzhao/GaLore)
- 2024/01 [KVQuant: Towards 10 Million Context Length LLM Inference with KV Cache Quantization (NIPS 2024)](https://github.com/SqueezeAILab/KVQuant)
- 2023/06 [AWQ: Activation-aware Weight Quantization for LLM Compression and Acceleration (MLSys 2024 Best Paper Award)](https://arxiv.org/abs/2306.00978)
- 2023/02 [DoRA: Weight-Decomposed Low-Rank Adaptation (ICML 2024 Oral)](https://github.com/nbasyl/DoRA)
- 2023/02 [Ten Lessons We Have Learned in the New “Sparseland”: A Short Handbook for Sparse Neural Network Researchers (ICLR 2023 Spotlight)](https://arxiv.org/abs/2302.02596)
- 2023/01 [SparseGPT: Massive Language Models Can Be Accurately Pruned in One-Shot (ICML 2023)](https://github.com/IST-DASLab/sparsegpt)

LLM
- 2024/05 [GRPO: Group Robust Preference Optimization in Reward-free RLHF (NIPS 2024)](https://arxiv.org/abs/2405.20304)
- 2024/09 [Gemma Scope: Open Sparse Autoencoders Everywhere All At Once on Gemma 2 (EMNLPW 2024)](https://arxiv.org/abs/2408.05147)
- 2024/04 [Rho-1: Not All Tokens Are What You Need (NIPS 2024 Best Paper Runner-up Award)](https://arxiv.org/abs/2404.07965)
- 2024/04 [Iterative RPO: Iterative Reasoning Preference Optimization (NIPS 2024)](https://arxiv.org/abs/2404.19733)
- 2024/04 [Layer Skip: Enabling Early Exit Inference and Self-Speculative Decoding (ACL 2024)](https://arxiv.org/abs/2404.16710)
- 2024/04 [Mixture-of-Depths: Dynamically allocating compute in transformer-based language models](https://arxiv.org/abs/2404.02258)
- 2024/03 [Gecko: Versatile Text Embeddings Distilled from Large Language Models](https://arxiv.org/abs/2403.20327)
- 2024/03 [RAFT: Adapting Language Model to Domain Specific RAG (COLM 2024)](https://arxiv.org/abs/2403.10131)
- 2024/02 [Echo Embeddings: Repetition Improves Language Model Embeddings (ICLR 2025)](https://github.com/jakespringer/echo-embeddings)
- 2023/12 [The Unlocking Spell on Base LLMs: Rethinking Alignment via In-Context Learning (ICLR 2024)](https://allenai.github.io/re-align/)
- 2023/10 [Controlled Decoding from Language Models (ICML 2024)](https://arxiv.org/abs/2310.17022)
- 2023/10 [Demystifying Embedding Spaces using Large Language Models (ICLR 2024)](https://arxiv.org/abs/2310.04475)
- 2023/08 [Linearity of Relation Decoding in Transformer Language Models (ICLR 2024 Spotlight)](https://arxiv.org/abs/2308.09124)

ETC
- 2024/10 [Differential Transformer (ICLR 2025 Oral)](https://arxiv.org/abs/2410.05258)
- 2024/10 [Restructuring Vector Quantization with the Rotation Trick (ICLR 2025 Oral)](https://arxiv.org/abs/2410.06424)
- 2024/03 [FaceXFormer : A Unified Transformer for Facial Analysis](https://arxiv.org/abs/2403.12960)
- 2024/03 [DiPaCo: Distributed Path Composition](https://arxiv.org/abs/2403.10616)
- 2024/02 ~~[Genie: Generative Interactive Environments (ICML 2024 Best Paper Award)](https://sites.google.com/view/genie-2024/home)~~
- 2024/01 [Depth Anything: Unleashing the Power of Large-Scale Unlabeled Data (CVPR 2024)](https://depth-anything.github.io/)
- 2023/12 [FuseMix: Data-Efficient Multimodal Fusion on a Single GPU (CVPR 2024 Highlight)](https://arxiv.org/abs/2312.10144)
- 2023/10 [Microscaling Data Formats for Deep Learning](https://arxiv.org/abs/2310.10537)

Good Article
- https://www.github.com/pytorch-labs/ao/issues/47
- https://deepmind.google/discover/blog/sima-generalist-ai-agent-for-3d-virtual-environments/
- https://pytorch.org/blog/accelerating-generative-ai-2/
- https://github.com/microsoft/unilm/tree/master/bitnet
- ~~https://huggingface.co/blog/embedding-quantization~~
- https://timdettmers.com/2023/01/30/which-gpu-for-deep-learning/
- https://www.ai21.com/blog/announcing-jamba
- https://sambanova.ai/blog/accurate-models-at-blazing-speed
- https://aws.amazon.com/ko/blogs/tech/klleon-digital-human-generative-model-aws-inferentia/
- https://github.com/google-research-datasets
- https://carpedm30.notion.site/AI-Compiler-Study-aaf4cff2c8734e50ad95ac6230dbd80b
- https://deepmind.google/discover/blog/tacticai-ai-assistant-for-football-tactics/
- ~~https://huggingface.co/docs/transformers/v4.15.0/parallelism~~
- [Google, ScaNN (Scalable Nearest Neighbors)](https://github.com/google-research/google-research/tree/master/scann)
- https://moon-walker.medium.com/%EB%A6%AC%EB%B7%B0-zero-infinity-breaking-the-gpu-memory-wall-for-extreme-scale-deep-learning-eeca3c035da5
- https://www.microsoft.com/en-us/research/blog/deepspeed-extreme-scale-model-training-for-everyone/
- https://thegenerality.com/agi
- https://sakana.ai/ai-cuda-engineer/
- https://diffusionflow.github.io/
- https://tulip-phalange-a1e.notion.site/a0a85f673c27473696512e06f580feff
- https://hanlab.mit.edu/blog/streamingllm
